### DISCUSSION TITLE
Building Map / Room Availability

### DISCUSSION THREAD

**Discussion by vivianaramos6** [uprm-inso4101-2024-2025-s1/semester-project-campus-resource-and-event-management#7] [Ideas]
The app could display the layout of the buildings' different floors and highlight available rooms and their student capacity.

**Comment by vivianaramos6** [comment]
I would also like to add that information such as types of table, projectors available, and whiteboards would also be useful.

**Comment by AngelUPR** [comment]
I think what we could do is after you select which building you want to see available, a list of all the available rooms are presented and after you click one, you get a preview of how the room looks and where it is. I think my problem with making a layout of the building would be that it would look conglomerated putting a big building with a lot of rooms into a screen (assuming this a mobile app)
